sdk/src/client/node_api/core/routes.rs

```diff
@@ -91,15 +91,18 @@ impl ClientInner {
             .await
     }
 
-    /// Checks if the account is ready to issue a block.
-    /// GET /api/core/v3/accounts/{accountId}/congestion
-    pub async fn get_account_congestion(&self, account_id: &AccountId) -> Result<CongestionResponse> {
-        let path = &format!("api/core/v3/accounts/{account_id}/congestion");
+    /// Return the information of committee members at the given epoch index. If epoch index is not provided, the
+    /// current committee members are returned.
+    /// GET /api/core/v3/committee/?epochIndex
```

Comment:
```suggestion
    /// GET /api/core/v3/committee?epochIndex
```

Reply:
This is not how urls are typically formatted

Reply:
Is it not? Something like https://api.testnet.shimmer.network/api/indexer/v1/outputs/basic?address=rms1qqh8nrvdrg08r2gzezmuaup35eygu7pzl7ekw6zx5vkc8j8gxwwnkvz8npl doesn't work anymore with an ending /

Reply:
Weird, it should not matter at all.

Reply:
The answer [here](https://stackoverflow.com/questions/1617058/ok-to-skip-slash-before-query-string) provides a pretty in-depth explanation, but the long and short is that it should not matter and I find it strange that the service only accepts one and not the other.

Reply:
Thanks. Yeah honestly I don't know why it's the case, but if I had to choose a flavour, I'd choose the one that works with our own stack, without the ending /. I'm resolving this anyway, feel free to address or not.